_posts/2017-07-26-property-based-testing-in-javascript.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title:  Property-based testing in JavaScript
+date: 2017-07-26 00:00:00 +0200
+categories: javascript testing
+author_name: Stefan Lau
+author_url : /author/stefanlau
+author_avatar: stefanlau
+show_avatar : true
+read_time : 10
+show_related_posts: false
+---
+
+As developers we are always looking for new ways to ensure that our code works as we would expected. One way to
+achieve this, which is not as commonly used as for example unit tests, is property-based testing. Property-based testing
+is a testing method based on (pseudo-)randomly generated input. It can be used at any level of the testing pyramid to
+validate some properties of algorithms or systems we built and is best used in addition to other tests, which test
+specific instances of inputs.
```

Comment:
You are already drawing some conclusions in the first paragraph of your post without validating them. Why can they be added "at any level of the testing pyramid" and why are they "best used in addition" ? Why can't they replace unit tests entirely? (I am not advocating that-  just asking the question).